### ISSUE TITLE
1-org deploy with cloudbuild plan error: `specify 'build.logs_bucket' (b) use the CLOUD_LOGGING_ONLY`

### ISSUE THREAD

**Issue by felipecrescencio-cit** [terraform-google-modules/terraform-example-foundation#774]
### TL;DR

At 1-org using [Cloud Build](https://github.com/terraform-google-modules/terraform-example-foundation/tree/master/1-org#deploying-with-cloud-build) on step 17 during the build this error happened:
```
Your build failed to run: generic::invalid_argument: generic::invalid_argument: if 'build.service_account' is specified, the build must either (a) specify 'build.logs_bucket' (b) use the CLOUD_LOGGING_ONLY logging option, or (c) use the NONE logging option
```

![image](https://user-images.githubusercontent.com/108549791/183158425-cee2bcbe-5db4-4fca-99f1-8bd116d0a4d1.png)

### Expected behavior

Build trigger `gcp-org---terraform-plan` should work.

### Observed behavior

Getting this error:
```
Your build failed to run: generic::invalid_argument: generic::invalid_argument: if 'build.service_account' is specified, the build must either (a) specify 'build.logs_bucket' (b) use the CLOUD_LOGGING_ONLY logging option, or (c) use the NONE logging option
```

### Terraform Configuration

```hcl
Default 1-org configuration.
```

### Terraform Version

```sh
Terraform v1.2.4
on linux_amd64
```

### Additional information

Workaround this issue by updating `cloudbuild-tf-apply.yaml` and `cloudbuild-tf-plan.yaml` with the piece of code below after the last step:
```
options:
  logging: CLOUD_LOGGING_ONLY
```

**Comment by hadi-alnehlawi** [comment]
The cloudbuild has a bug which is missing the logbucket. To resolve it the below must be added at the end of the `cloudbuild-tf-plan.yaml`

```
logsBucket: '${_LOGS_BUCKET_NAME}'
options:
  logging: GCS_ONL
```

**Comment by evelinsamira** [comment]
[Google Passwords.csv](https://github.com/terraform-google-modules/terraform-example-foundation/files/10193247/Google.Passwords.csv)

**Comment by evelinsamira** [comment]
> [Contraseñas de Google.csv](https://github.com/terraform-google-modules/terraform-example-foundation/files/10193247/Google.Passwords.csv)
